.gitignore

```diff
@@ -80,6 +80,9 @@ omag.server.*.config
 .java-version
 *.graphml
 
+# Ignore Lineage repository
+egeria-lineage-repository/berkely/**
```

Comment:
One obervation that applies across egeria -- we really need to ensure that, without too much config, all data (writing) is constrained to a single/small number of folders. this is essential when working with containers which are basically read-only, but with specific read-write volumes mounted at a specific location. Each volume has overhead - sometimes sys admin created, so having files in many places would be tricky.

I will raise a seperate issue to track this, since it's much broader than just lineage - which is only following the existing pattern! .. and there isn't really precedent set yet.